test/test-v8debugapi.js

```diff
@@ -43,6 +45,67 @@ function stateIsClean(api) {
   return true;
 }
 
+function validateVariable(variable) {
+  if (variable.name) {
+    assert.equal(typeof variable.name, 'string');
+  }
+  if (variable.value) {
+    assert.equal(typeof variable.value, 'string');
+  }
+  if (variable.type) {
+    assert.equal(typeof variable.type, 'string');
+  }
+  if (variable.members) {
+    variable.members.forEach(validateVariable);
+  }
+  if (variable.varTableIndex) {
+    assert.ok(Number.isInteger(variable.varTableIndex) &&
```

Comment:
Specifically, I'm curious about the choice of value for MAX_INT. Is 2^31-1 the upper bound based on proto definition?